src/layouts/sidebar-sidecar/index.tsx

```diff
@@ -82,6 +83,12 @@ const SidebarSidecarLayoutContent = ({
     },
   }
 
+  /**
+   * Handles the top positioning of the sidebar if a user has scrolled
+   * and prevents scrolling on the rest of the page body
+   */
```

Comment:
🚧 Since this hook only affects `document.body.style.overflow`, can this comment be updated to reflect that?

```suggestion
  /**
   * Prevents scrolling on the rest of the page body
   */
```

Reply:
Ahh good catch, ty 😄